pnet_sys/src/unix.rs

```diff
@@ -28,7 +28,46 @@ pub mod public {
     pub const IP_HDRINCL: libc::c_int = libc::IP_HDRINCL;
 
     pub const IFF_LOOPBACK: libc::c_int = libc::IFF_LOOPBACK;
+    
+    #[cfg(target_os = "macos")]
+    mod device_flags {
+        use super::libc;
+        // XNU
+        /// https://github.com/aosm/xnu/blob/master/bsd/net/if.h#L140-L156
+        pub const IFF_UP: libc::c_int          = 0x1;       //  interface is up
+        pub const IFF_BROADCAST: libc::c_int   = 0x2;       //  broadcast address valid
+        pub const IFF_DEBUG: libc::c_int       = 0x4;       //  turn on debugging
+        // pub const IFF_LOOPBACK: libc::c_int    = 0x8;       //  is a loopback net
```

Comment:
Is there a reason this is commented out?

Reply:
Yes. It's defined outside `device_flags` and set to the value exported by `libc`.

Reply:
But then again, it could be left out completely here, not just be commented out.

Reply:
I think I'd like it left in, with a comment saying where it's actually defined. Or alternatively, have it be an alias for the one in the standard lib. Or specify it here in addition to libc. I'm happy either way.

Reply:
It's defined just a few lines of code above here. And it can't be defined here as well, since all these constants are exported out and that would collide.